### PR TITLE
SSH Version of the IPS

### PR DESCRIPTION
**Pull request by malvads**
* This is the ssh version of the ips

### CODE REVIEW

**Review by ljblancoredborder — CHANGES_REQUESTED** [review]

File: resources/bin/rb_get_sensor_rules.sh
[the remark this answers is not in the document]
Comment (malvads) [reply resources/bin/rb_get_sensor_rules.sh:1]: Not from this task @ljblancoredborder

File: resources/scripts/rb_associate_sensor.rb [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+end
+
+
+opt = Getopt::Std.getopts("u:p:i:m:h")
+
+if opt["h"]
+  usage
+end
+
+if opt["u"].nil? || opt["p"].nil? || opt["i"].nil? || opt['m'].nil?
```
[the remark this answers is not in the document]
Comment (malvads) [reply resources/scripts/rb_associate_sensor.rb:36]: Not from this task @ljblancoredborder , centos6 script

File: resources/lib/rb_wiz_lib.rb [hunk 2]
```diff
@@ -810,6 +810,187 @@ def doit
 
 end
 
+class ModeConf < WizConf
```
[the remark this answers is not in the document]
Comment (malvads) [reply resources/lib/rb_wiz_lib.rb:813]: Following current wiz structure @ljblancoredborder changes would not be made

File: resources/lib/rb_config_utils.rb [hunk 3]
```diff
@@ -32,6 +37,147 @@ def self.get_encrypt_key(password)
         ret
     end
 
+    # Resolve manager dns name to ip
+    def self.managerToIp(str)
+      ipv4_regex = /\A(\d{1,3}\.){3}\d{1,3}\z/
```
[the remark this answers is not in the document]
Comment (malvads) [reply resources/lib/rb_config_utils.rb:42]: Working as intended @ljblancoredborder not gonna change

**Pull request merged**